components/pluralrules/Cargo.toml

```diff
@@ -16,26 +16,20 @@ include = [
 
 [dependencies]
 icu-locale = { path = "../locale" }
-serde = { version = "1.0", optional = true, features = ["derive"] }
-serde_json = {version = "1.0", optional = true }
-bincode = { version = "1.3", optional = true }
-serde-tuple-vec-map = { version = "1.0", optional = true }
+icu-data-provider = { path = "../data-provider" }
+icu-cldr-json-data-provider = { path = "../cldr-json-data-provider", optional = true }
```

Comment:
Feature crates should depend on the trait definitions (`icu-data-provider`) but not any particular implementation (like `icu-cldr-json-data-provider`).

Where are you using `icu-cldr-json-data-provider` other than in test code?